common/strings/patch.h

```diff
@@ -147,6 +147,8 @@ struct HunkIndices {
   }
   bool operator!=(const HunkIndices& other) const { return !(*this == other); }
 
+  std::string ToString() const;
```

Comment:
something named `ToString()` sounds a bit like a DebugString, but here it is actually formatted as expected in a patch, so I'd try to see if there can be a better name for this.
(Or even: remove it entirely and keep it with the original `operator<<` implementation).

Anyway, not a biggy, but if you have a good idea for different naming, this can also be done in a later PR.

Reply:
I agree naming is not great. The need for the separate function comes from 
```c++
std::ostream& operator<<(std::ostream& stream, const HunkHeader& header) {
  return term::Colored(
      stream,
      absl::StrCat("@@ -", header.old_range.ToString(), " +",
                   header.new_range.ToString(), " @@", header.context),
      term::Color::kGreen);
}
```
I don't see a way around having that explicit "ToString" function because we require term::Colored to receive strings. Perhaps something like: Format, FormatToString is clearer?

Reply:
Yes, maybe `FormatToString()` would be nicer (it has less of the 'this is a debug-output' vibe).